### DISCUSSION TITLE
Include and command errors during configuring

### DISCUSSION THREAD

**Discussion by nmartonb** [sofa-framework/sofa#4825] [Build / Config / Environment] [answered]
Hello!
I tried to build SOFA for the first time and I get the following errors during configuring. I found the SofaMacros.cmake file in sofa/src/framework/Config/cmake, but apparently CMake can not include it. I would appreciate any help with this.

![SofaCmakeErrors](https://github.com/sofa-framework/sofa/assets/80408411/25837ff4-72b2-492e-8ca3-33a16f810051)

**Comment by alxbilger** [comment]
First time I see this error. Have you changed some CMake properties? Does it make a difference to select `Visual Studio 17 2022` as the generator, instead of ninja?

**Comment by nmartonb** [comment, marked as answer]
Changing the generator did not help. I deleted my local repo and cloned it again, then I reinstalled CMake, now it seems to be working. Thanks for the help.
